.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
I need clarification about the difference between this section and the next. How are they different? Can you elaborate?

When you describe the issue, aren't you inherently also describing your use case scenario?

Reply:
Good point. Removed the "Is your request..." section